fundamentals/points.md

```diff
@@ -68,6 +104,9 @@ layer : napari.layers.Points
 
 The input data to the points layer must be an NxD numpy array containing the coordinates of N points in D dimensions. The ordering of these dimensions is the same as the ordering of the dimensions for image layers. This array is always accessible through the `layer.data` property and will grow or shrink as new points are either added or deleted.
 
+## points properties
```

Comment:
Should this maybe be `Using the points properties dictionary`. It's such amazing functionality, but I worry it's getting lost in the long list of everything else

Reply:
I think you should also add something about the points properties dictionary to the top section at the end of this paragraph

> The points layer allows you to display an NxD array of N points in D coordinates. You can adjust the size, face color, edge color of all of the points independently. You can also adjust the opactiy, edge width, and symbol representing all the points simultaneously.

Just to really emphasize this functionality

Reply:
I'd also add a link here to your applications note, maybe with a comment like "to see this in action look here" etc

Reply:
Great points! I have implemented your suggestions.